style.css

```diff
@@ -0,0 +1,102 @@
+@media screen and (max-width: 375px) {
+  body {
+    margin-left: 0.1px;
```

Comment:
Please update your media query to at least 768px 😉. Using 375px will make your styling appear only on the iPhone SE or similar screens, while on almost all other devices you won't have any styling:

Your project on Iphone XR:
![image](https://user-images.githubusercontent.com/42799579/205207980-d380578d-a210-4ed8-8f5b-00c253e79527.png)

Reply:
Fixed scrren size for 768px